core/modules/system/config/system.block_style.simple.json

```diff
@@ -0,0 +1,16 @@
+{
+    "_config_name": "menu.block_style.simple",
+    "simple_absolute": {
+        "1": "1"
+    },
+    "simple_disable_mouse_events": {
+        "1": "0"
+    },
+    "simple_remove_attributes": {
+        "1": "1"
+    },
+    "simple_css_selectors": "#main-menu",
```

Comment:
I'm not sure how this is being used, but it'd be best to avoid ID selectors in CSS. Is there a class? `.block-menu-main-menu` or something like that?